_pages/en_US/formatting-sd-(windows).txt

```diff
@@ -0,0 +1,46 @@
+---
+title: "Formatting SD (Windows)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This is an add-on section for formatting an SD card to work with the 3DS.
+
+If the 3DS already recognizes the SD card, this guide is not required.
+
+This page is for Windows users only. If you are not on Windows, check out the [Formatting SD (Linux)](formatting-sd-(linux)) or [Formatting SD (Mac)](formatting-sd-(mac)) pages.
+
+### What You Need
+
+* The latest version of [guiformat](http://www.ridgecrop.demon.co.uk/index.htm?guiformat.htm)
+
+### Instructions
+
+1. Insert your SD card into your computer
+1. If the SD card has any files and folders on it, copy everything to a folder on your computer
+1. Run `guiformat.exe`
+1. Select your SD card's drive letter for "Drive"
+
+	Make sure you choose the correct drive letter, otherwise you might accidentally erase the wrong drive!
+	{: .notice--danger}
+
+1. Select a size for "Allocation unit size"
+  + If the SD card is 32GB or below, choose 32768
+  * If the SD card is above 32GB, choose 65536
```

Comment:
Folks in discord were saying this threshold was a 128GB SD card -- if it's 32GB, and you're certain of that, let me know and I'll update #1772

Reply:
Yes I'm pretty sure it has usually been 64k size for any cards above 32gb, though 64gb with 32k has usually been fine I think.